packages/php-wasm/compile/Dockerfile

```
@@ -1055,5 +1101,12 @@ RUN \
         rm /root/output/php.js && \
         \
         # Rename the build files to their final names
-        mv /root/output/php-module.js "/root/output/$JS_FILENAME" && \
-        mv /root/output/php.wasm "/root/output/$WASM_FILENAME"
\ No newline at end of file
+        mv /root/output/php-module.js "/root/output/$JS_FILENAME"; \
+        mkdir -p /root/output/${PHP_VERSION_ESCAPED}/; \
+		mv /root/output/php.wasm "/root/output/${PHP_VERSION_ESCAPED}/$WASM_FILENAME"; \
+		if [ "${WITH_SOURCEMAPS}" = "yes" ]; then \
+			sed -i 's/..\/php-src/php-src/g' /root/output/php.wasm.map; \
```

Let's remove the `.git` directory:

```suggestion
			rm -rf /root/php-src/.git; \
			sed -i 's/..\/php-src/php-src/g' /root/output/php.wasm.map; \
```

Preserving it triggers the following error on my end:

```
Running docker run --name php-wasm-tmp --rm -v packages/php-wasm/web/public:/output php-wasm sh -c cp -rf /root/output/* /output && mkdir -p /output/terminfo/x  ...
cp: cannot create regular file '/output/7_4_33/php-src/.git/objects/pack/pack-7063960767cc0b8bc15af3550492d4c4145a5efa.idx': Permission denied
```